gitops/argocd/argo-apps/pvc-cleaner.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: pvc-cleaner
+  namespace: openshift-gitops
+
+spec:
+  destination:
+    namespace: pvc-cleaner
```

Comment:
Hey @AndrienkoAleksandr, I'm not sure about installing `pvc-cleaner` application in `pvc-cleaner` namespace. AFAIK, gitops have limited accessibility to other namespaces as per current design. In addition, following the current pattern, Argo CD applications are being installed in `openshift-gitops` ns. I would ask for @bnallapeta  's opinion on this.

Reply:
I guess to have wide access to the another namespaces for some cases we have permission workaronds https://github.com/openshift-pipelines/pipelines-service/tree/main/exploration/argocd-external/gitops/sre/environment/compute/base/argocd-rbac. For example https://github.com/openshift-pipelines/pipelines-service/blob/main/exploration/argocd-external/gitops/sre/environment/compute/base/argocd-rbac/argo-syncer-role.yaml and https://github.com/openshift-pipelines/pipelines-service/blob/main/exploration/argocd-external/gitops/sre/environment/compute/base/argocd-rbac/argo-tekton-role.yaml ... But I agree, that's interesting to get feedback from @bnallapeta

Reply:
@AndrienkoAleksandr The link you shared is in the exploration folder for now. We wish to use this when we get to a state where one single Argo CD server is managing multiple applications from multiple clusters. 
But as of today, we are installing Argo CD server on each physical cluster and thus using the openshift-gitops namespace. Please change the namespace for the pvc-cleaner accordingly.

Reply:
Done.

Reply:
> Argo CD applications are being installed in openshift-gitops ns.

Hey @AndrienkoAleksandr, I meant this `pvc-cleaner` Argo CD application to have `spec.destination.namespace:openshift-gitops`.
-> You can keep the Argo CD application destination namespace separate from the namespace where the application/operator controller itself will reside. This is also something that we're following for other operators as well like - [Openshift Pipelines](https://github.com/openshift-pipelines/pipelines-service/blob/main/gitops/argocd/tektoncd/openshift-operator.yaml#L5)
-> Please revert to the desired namespace for pvc-cleaner where `pvc-cleaner` controller will reside.  
Sorry for the confusion.